addons/project/views/project_views.xml

```diff
@@ -245,6 +245,7 @@
             <field name="res_model">project.task.type</field>
             <field name="view_mode">tree,kanban,form</field>
             <field name="view_id" ref="task_type_tree"/>
+            <field name="domain">[('user_id', '=', False)]</field>
```

Comment:
When we add this domain, how do we see the stages with a user set? 🤔 
Is it possible to see all stages in another menu/action?

Reply:
@xavierbol , Agree with you. There is no other menu to see the personal stages. So I have just added the filter and applied the default filter when we open the stages menu. 
Done all other changes.

cc @aju-odoo 

Thanks